docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -7,61 +7,40 @@ token which is attached to the end of a url, digitally signed and authorised by
-    A reference number used to represent the collection exercise inside the ONS
-  period_id
-    A numerical reference to either a month or quarter time period
-  period_str
-    A display name for the period referenced above
-  ref_p_start_date
-    A start date for the relevant reference period
-  ref_p_end_date
-    An end date for the relevant reference period
-  employment_date
-    A specific date for the questionnaire to display to the respondent.
-  form_type
-    The particular form_type for a responding unit
-  survey_url
-    An optional URL for a remote survey JSON. This claim is used to tell Survey Runner to load the schema JSON from a remote location
-  return_by
-    A date which represents the return date for a particular collection exercise for a survey. Represented by a ISO_8601 YYYY-MM-DD date.
-  roles
-    An array of roles that this token is allowed to assume. This parameter is currently optional, EQ will still work if it is not passed.
-  region_code
-    Region code identifier, used to change content for different regions. Format ISO 3166-1 alpha-2 (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) e.g. GB-GBN. This parameter is currently optional, EQ will still work if it is not passed
-  language_code
-    Language code identifier, used to change language displayed. Format as per ISO-639-1 (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) e.g. "en" for English; "cy" for Welsh. This parameter is currently optional; the default is "en".
-  variant_flags
-    JSON object containing name:value pairs describing variations in the questionnaire e.g. `{"flag_1": true, "flag_2": false}`. The only value type allowed is boolean i.e. `true` or `false`.
-
-* All dates are represented in ISO_8601 and are assumed to be UTC unless a timezone element is supplied.
-* All elements not indicated as optional are required as part of the claim.
+Schema Definition - Required Fields
+===================================
+The following metadata keys are always required for the survey runner, they do not appear in individual survey metadata definitions.
 
+``tx_id``
+  see :doc:`jwt_profile`
+``iat``
+  JWT Issued At claim, see https://tools.ietf.org/html/rfc7519#section-4.1.6
+``exp``
+  JWT Expiration Time claim, see https://tools.ietf.org/html/rfc7519#section-4.1.4
+``ru_ref``
+  The responding unit reference id - with checkletter appended.
+``eq_id``
+  The eQ questionnaire instance id.
+``collection_exercise_sid``
+  A reference number used to represent the collection exercise inside the ONS
+``form_type``
+  The particular form_type for a responding unit
+``ru_name``
+  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present.
+``trad_as``
+  The trading as name for a responding unit. Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present.
+``variant_flags``
```

Comment:
I think we should remove `variant_flags` from runner?
I think they worked in the days before we could define metadata, but now its just like a black hole.
Thoughts?

Reply:
Well it looks like they're only used to pass validation, but nowhere else in the code!

Reply:
Raised https://github.com/ONSdigital/eq-survey-runner/pull/1715